packages/block-library/src/accordion-group/index.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Server-side rendering of the `core/accordion-group` block.
+ *
+ * @package WordPress
+ * @since 6.6.0
+ *
+ * @param array $attributes The block attributes.
+ * @param string $content The block content.
+ *
+ * @return string Returns the updated markup.
+ */
+function render_block_core_accordion_group( $attributes, $content ) {
+	if ( ! $content ) {
+		return $content;
+	}
+
+	$suffix = wp_scripts_get_suffix();
+	if ( defined( 'IS_GUTENBERG_PLUGIN' ) && IS_GUTENBERG_PLUGIN ) {
+		$module_url = gutenberg_url( '/build/interactivity/accordionGroup.min.js' );
+	}
+
+	wp_register_script_module(
+		'@wordpress/block-library/accordion-group',
+		isset( $module_url ) ? $module_url : includes_url( "blocks/accordion-group/view{$suffix}.js" ),
+		array( '@wordpress/interactivity' ),
+		defined( 'GUTENBERG_VERSION' ) ? GUTENBERG_VERSION : get_bloginfo( 'version' )
+	);
+
+	wp_enqueue_script_module( '@wordpress/block-library/accordion-group' );
+
+	$p         = new WP_HTML_Tag_Processor( $content );
+	$autoclose = $attributes['autoclose'] ? 'true' : 'false';
+
+	if ( $p->next_tag( array( 'class_name' => 'wp-block-accordion-group' ) ) ) {
+		$p->set_attribute( 'data-wp-interactive', 'core/accordion' );
+		$is_open = wp_json_encode( block_core_accordion_group_item_ids() );
+		$p->set_attribute( 'data-wp-context', '{ "autoclose": ' . $autoclose . ', "isOpen":' . $is_open . ' }' );
+
+		// Only modify content if directives have been set.
+		$content = $p->get_updated_html();
+	}
+
+	return $content;
+}
+
+/**
+ * Returns the existing list of ids, or adds a new id to the list.
+ *
+ * @since 6.6.0
+ *
+ * @param string $id The id of the accordion item.
+ * @return array|void Returns the ids for the  or nothing.
+ */
+function block_core_accordion_group_item_ids( $id = null ) {
```

Comment:
@gziolo Could you help us with the name of this function? How is it supposed to be named so that it can be used from the other block? What pattern could be used here?